packages/core/src/utils/snapshot-progress-renderer.ts

```diff
@@ -0,0 +1,90 @@
+import { Container, Contracts } from "@arkecosystem/core-kernel";
+import { SnapshotApplicationEvents } from "@arkecosystem/core-snapshots";
+import { Ora } from "ora";
+
+export class ProgressRenderer {
+    private isAnyStarted: boolean = false;
+
+    private spinner: Ora;
+
+    private blocksProgress: string = "---.--";
+    private transactionsProgress: string = "---.--";
+    private roundsProgress: string = "---.--";
+
+    // @ts-ignore
+    private blocksCount: number = 0;
+    // @ts-ignore
+    private transactionsCount: number = 0;
+    // @ts-ignore
+    private roundsCount: number = 0;
+
+    // private interval?: NodeJS.Timeout;
+
+    public constructor(spinner: Ora, kernelApp: Contracts.Kernel.Application) {
```

Comment:
Rename `kernelApp` to `app` because `kernelApp` makes it sound like there are different types of applications that could be injected, which is not the case.